akka-persistence/src/main/scala/akka/persistence/Eventsourced.scala

```diff
@@ -546,6 +546,7 @@ private[persistence] trait Eventsourced extends Snapshotter with Stash with Stas
           updateLastSequenceNr(p)
           try {
             peekApplyHandler(p.payload)
+            onPersistSuccess(p.payload)
```

Comment:
Another thought (sorry, just at my notebook in random short bursts): since persists succeeded, yet the handler may have thrown, technically we still should have gotten the onPersistSuccess callback, since persistence succeeded, and then the handler failed, so slightly different no?